addon/components/horizontal-bar-chart.js

```diff
@@ -40,6 +42,63 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * @override
+  * Overrides values in addon/mixins/axis-titles.js
+  */
+  xAxisPositionX: Ember.computed('graphicWidth', 'labelWidthOffset', function(){
+    return this.get('graphicWidth') / 2;
+  }),
+
+  /**
+  * @override
+  * X Axis Titles needs some extra padding or else it will intersect with the lowest bar
+  * TO-DO: Change the 20px padding to be a variable that can be referenced elsewhere
+  */
+  xAxisPositionY: Ember.computed('graphicBottom', 'axisPadding', function(){
+    return this.get('graphicBottom') + this.get('xAxisPadding');
+  }),
+
+  /**
+  * @override
+  */
+  yAxisPositionX: Ember.computed('graphicBottom', 'axisPadding', function(){
+    return -(this.get('graphicHeight') / 2);
+  }),
+
+  /**
+  * @override
+  */
+  yAxisPositionY: Ember.computed('labelWidthOffset', function(){
+    return -(this.get('labelWidthOffset'));
+  }),
+
+  /**
+  * @override
+  * Horizontal Bar Charts require that marginLeft be dependent on 'horizontalMargin'
+  * Otherwise, without override will result in marginLeft ~= 0 (axis location will always be flush on the left)
+  */
+  marginLeft: Ember.computed('hasAxisTitles', 'horizontalMarginLeft', 'horizontalMargin', function(){
+     return this.get('hasAxisTitles') ? this.get('horizontalMarginLeft') + this.get('horizontalMargin'): this.get('horizontalMargin');
```

Comment:
Break this line as it's too long. We try to keep at most 80 characters per line.